src/Microsoft.Azure.WebJobs.Extension.Storage/Blobs/StorageBlobExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage;
+
```

Comment:
Remove this type - StorageBlob doesn't even exist anymore, right?

Reply:
Done